### ISSUE TITLE
Resolve same-named packages

### ISSUE THREAD

**Issue by mrakitin** [bluesky/bluesky-kafka#5]
I was confused by the same-named packages. One, which was originally packaged with conda was from https://github.com/NSLS-II/bluesky-kafka, and the new repo https://github.com/bluesky/bluesky-kafka, which is the source of https://pypi.org/project/bluesky-kafka/0.1.0/#files. This confusion resulted in a CI packaging error: https://dev.azure.com/nsls2forge/nsls2forge/_build/results?buildId=3091&view=logs&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=6d4b912b-175d-51da-0fd9-4d30fe1eb4e7&l=1104.

https://github.com/NSLS-II/nslsii/blob/8c5ccb2fc7416387b46f64d5f286de0a77162cda/nslsii/__init__.py#L14 shows uses the interface of the old library. I think it should be reworked before we massively deploy. Was the `kafka.py` missing from the new repo? Also, the old repo should be archived so it's not used mistakenly (I'll do it now).

@jklynch, let's resolve it tomorrow.

**Comment by jklynch** [comment]
There is no `kafka.py` in the new repo. Everything is in `__init__.py`.
I think both repos define a `Publisher` (because that is the bluesky way!).
nslsii works correctly if bluesky/bluesky-kafka v0.1.0 is installed.

**Comment by mrakitin** [comment]
Yes, I noticed that. My problem was that conda prefered to pick the old version v0.1, not the new one. I changed the label to `obsolete` for the old p.ackage, and that helped to solve the issue: https://anaconda.org/nsls2forge/bluesky-kafka/files
![image](https://user-images.githubusercontent.com/13209176/85814942-321afa80-b735-11ea-93e2-3cbe9b38da93.png)

I think there is nothing else we can do here. Thanks for the clarification. Closing
